packages/components/src/internal/components/entities/ParentEntityEditPanel.tsx

```diff
@@ -285,9 +306,14 @@ export class ParentEntityEditPanel extends Component<Props, State> {
     };
 
     onAddParent = (): void => {
-        this.setState(state => ({
-            currentParents: state.currentParents.push({ type: undefined, value: undefined, ids: undefined }),
-        }));
+        this.setState(state => {
+            const toAdd = [{ type: undefined, value: undefined, ids: undefined }];
+            if (state.currentParents.size == 0)
+                toAdd.push({ type: undefined, value: undefined, ids: undefined })
```

Comment:
I'm not sure I am following this. Is it saying that when the `currentParents.size == 0` then we want to push 2 new values to the `currentParents`?